### ISSUE TITLE
Warning: componentWillUpdate is deprecated and will be removed...

### ISSUE THREAD

**Issue by jaredegan** [n4kz/react-native-material-textfield#183]
![image](https://user-images.githubusercontent.com/303795/66680563-d3612f80-ec3e-11e9-875e-ce89847c585e.png)

Version Info:
- React Native version 0.60.5
- React version 16.8.6

I'm two YellowBox warnings from your library:
"Warning: componenetWillUpdate is deprecated and will be removed in the next major version"
and
"Warning: componenetWillReceiveProps is deprecated and will be removed in the next major version"

The official React guide on the first warning is here: https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html

Meanwhile I'm suppressing these warnings with the following code, include in my initial App.tsx file:

```javascript
import { YellowBox } from 'react-native';
YellowBox.ignoreWarnings(['Warning: componentWillUpdate', 'Warning: componentWillReceiveProps']);
```

**Comment by n4kz** [comment]
Thanks for issue! I've just released `0.13.0` with this warnings fixed.

**Comment by jaredegan** [comment]
Appreciate the speedy fix!
